master/buildbot/test/unit/test_schedulers_forcesched.py

```diff
@@ -388,9 +388,9 @@ def do_ParameterTest(self,
             if gotSpec != expectSpec:
                 try:
                     import xerox
-                    formated = self.formatJsonForTest(gotJson)
+                    formatted = self.formatJsonForTest(gotJson)
                     print(
-                        "You may update the test with (copied to clipboard):\n" + formated)
+                        "You may update the test with (copied to clipboard):\n" + formatted)
                     xerox.copy(formated)
```

Comment:
formated here also (see the pylint and flake8 errors)

Reply:
@tardyp thanks, can you share what I need to run to check pylint and flake8?  any specific flags?

Reply:
you can just run ```make virtualenv``` and then ```make flake8``` and ```make pylint```

Or you can push the code, and wait for our CI to tell you the results:

![image](https://user-images.githubusercontent.com/109859/34639526-d6d4dc1e-f2e1-11e7-90e5-d1f5fdc69229.png)